discussion/src/main/java/org/openedx/discussion/presentation/threads/DiscussionThreadsFragment.kt

```diff
@@ -722,36 +722,37 @@ private fun DiscussionThreadsScreenTabletPreview() {
 }
 
 private val mockThread = org.openedx.discussion.domain.model.Thread(
-    "",
-    "",
-    "",
-    "",
-    "",
-    "",
-    "",
-    TextConverter.textToLinkedImageText(""),
-    false,
-    true,
-    20,
-    emptyList(),
-    false,
-    "",
-    "",
-    "",
-    "",
-    DiscussionType.DISCUSSION,
-    "",
-    "",
-    "Discussion title long Discussion title long good item",
-    true,
-    false,
-    true,
-    21,
-    4,
-    false,
-    false,
-    mapOf(),
-    10,
-    false,
-    false
-)
\ No newline at end of file
+    id = "",
```

Comment:
`Prefer mutableIntStateOf instead of mutableStateOf` at line # 241

```
    val firstVisibleIndex = remember {
        mutableStateOf(scrollState.firstVisibleItemIndex)
    }
```